### PR TITLE
Introduce additional textlint rules

### PR DESCRIPTION
**Pull request by svrnm**
@chalin please take a look, hope this ruleset is better than the default

### CODE REVIEW

File: .textlintrc.json [hunk 1]
```diff
@@ -27,34 +27,87 @@
...
+        "JMX",
         "K8s",
+        "Kafka",
         "KubeCon",
         "Kubernetes",
+        "MacBook",
+        "macOS",
         "MySQL",
         "NGINX",
+        "npm",
```
Comment (chalin) [inline .textlintrc.json:60]: NPM is a valid acronym, as we can gather from this Node.js page: https://nodejs.dev/en/learn/an-introduction-to-the-npm-package-manager/

```suggestion
        "NPM",
```

Comment (svrnm) [reply .textlintrc.json]: Oh, thanks! I removed it!

File: content/en/blog/2022/troubleshooting-nodejs.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ For most users, the following should cover it:
 
 ```js
 // First run `npm install @opentelemetry/instrumentation-foo @opentelemetry/instrumentation-bar
```
Comment (chalin) [inline content/en/blog/2022/troubleshooting-nodejs.md:80]:
```suggestion
// First run: npm install @opentelemetry/instrumentation-foo @opentelemetry/instrumentation-bar
```

**Review by chalin — APPROVED** [review]
Thanks for cherry-picking all those rules.
LGTM once checks pass.

**Pull request merged**